flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogHiveMetadataTest.java

```diff
@@ -266,7 +266,7 @@ void testCreateTableWithConstraints() throws Exception {
                         Arrays.asList(
                                 Column.physical("x", DataTypes.INT().notNull()),
                                 Column.physical("y", DataTypes.TIMESTAMP(9).notNull()),
-                                Column.physical("z", DataTypes.BIGINT().notNull())),
```

Comment:
Seems it's changed in this [pr](https://github.com/apache/flink/pull/21522/files#diff-9725c1745d0c1aa78f5bfa75c771a175ee68576bb7ee0e8400cd8fa616e0aa9bR269) by mistake.

Reply:
I'm curious why the CI of that PR succeed?

Reply:
`Assume.assumeTrue(HiveVersionTestUtil.HIVE_310_OR_LATER);`
The test only runs when the it's hive3+. The ci of pr won't run with Hive3. It only runs with Hive3 nightly.

Reply:
Reasonable